alf/algorithms/rl_algorithm.py

```diff
@@ -235,11 +239,14 @@ def __init__(self,
         self._current_time_step = None
         self._current_policy_state = None
         self._current_transform_state = None
-
+        self._cached_exp = []  # for lazy observation
         if self._env is not None and not self.on_policy:
             replay_buffer_length = adjust_replay_buffer_length(
                 config, self._num_earliest_frames_ignored)
 
+            if self._episodic_annotation:
+                assert self._env.batch_size == 1, "only support non-batched environment"
```

Comment:
Add this to the docstring of ``episodic_annotation``?

Reply:
The assertion is not necessary here so remove also